docs/guides/aws_provider_s3_stackit.md

```diff
@@ -0,0 +1,91 @@
+---
+page_title: "Using AWS Provider for STACKIT S3"
+---
+# Using AWS Provider for STACKIT S3
+
+## Overview
+
+This guide outlines the process of utilizing the [AWS Terraform Provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs) alongside the STACKIT provider to create and manage STACKIT S3 ressources.
+
+## Steps
+
+1. **Configure STACKIT Provider**
+
+    First, configure the STACKIT provider to connect to the STACKIT services.
+
+    ```hcl
+    provider "stackit" {
+      region = "eu01"
+    }
+    ```
+
+2. **Define STACKIT S3 Bucket**
+
+    Create a STACKIT S3 Bucket and obtain credentials for the AWS provider.
+
+    ```hcl
+    resource "stackit_objectstorage_bucket" "example" {
+      project_id = ""
```

Comment:
I think here would be more clear to use `"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"`, also to be aligned with the other guides

Reply:
Maybe even better, `PROJECT_ID` (if you opt for this one please also update it in the other guides

Reply:
Created a new commit, containing the changes.